Resources/Computing/ComputingElement.py

```diff
@@ -428,7 +428,8 @@ def getDescription( self ):
           tagList = []
           for i in range( 2, cores+1 ):
             tagList.append( '%dCore' % i )
-          ceDict['Tag'] = tagList  
+          ceDict.setdefault( 'Tag', [] )
```

Comment:
:D I agree that sometimes it is useless, but the reason I asked here is that I did not understand what you were trying to do in a first place. It's only after I looked at the whole file that I understood that there can already be a a value for tag, and that you create it only if it does not exist with this set defaut. So more than compressing to a maximum here (which I don't like either :-) ), I found it more comprehensible. But maybe it's only me :-)